universal-application-tool-0.0.1/app/forms/translation/MultiOptionQuestionTranslationForm.java

```diff
@@ -0,0 +1,52 @@
+package forms.translation;
+
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import services.LocalizedStrings;
+import services.question.QuestionOption;
+import services.question.exceptions.UnsupportedQuestionTypeException;
+import services.question.types.MultiOptionQuestionDefinition;
+import services.question.types.QuestionDefinition;
+import services.question.types.QuestionDefinitionBuilder;
+
+public class MultiOptionQuestionTranslationForm extends QuestionTranslationForm {
```

Comment:
praise: Nice work with QuestionTranslationForm and this extension! This makes a lot of sense to me.

Reply:
Thanks Natalie! I modeled it off your work on the QuestionForms